app-editors/pyvim/pyvim-3.0.3.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..11} )
+DISTUTILS_USE_PEP517=setuptools
+inherit distutils-r1 edo
+
+COMMIT="14118ad2e4d0da2e955fd9069b8772408307618b"
+DESCRIPTION="An implementation of Vim in Python"
+HOMEPAGE="https://pypi.org/project/pyvim/ https://github.com/prompt-toolkit/pyvim"
+SRC_URI="https://github.com/prompt-toolkit/${PN}/archive/${COMMIT}.tar.gz -> ${P}.gh.tar.gz"
```

Comment:
I think you could drop the `.gh` part of the file extension.

Reply:
Doesn't pkgcheck require this now?

Reply:
You are right @thesamesam (https://github.com/pkgcore/pkgcheck/pull/383) sorry for the noise @CyberTailor! TIL.